collect_app/src/main/res/layout/help_text_layout.xml

```diff
@@ -53,5 +53,6 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:visibility="gone"
+            android:textColor="?warningColor"
```

Comment:
If it's red in both themes it doesn't really make sense to do that in that way, why don't you just use `@color/red` here?

Reply:
@grzesiek2010 I see your point. My thinking in this matter is as follows:

1. The fact we can use the same color (`@color/red`) for warning in both themes is a coincidence and not a rule or design decision
2. Using attribute allows us to name the color according to its use, rather than value "red". So using `?warningColor` is a nice way to create an alias that is defined by usage not value. (alternatively we can create a global color such as `@color/warningColor`
3. Warning color depends on the theme. It depends on the same looks of theme as `?primaryTextColor`, `?colorAccent`. So theme is the context of the color, and the reason it has this value, so it seems like a natural place to put it. 

WDYT?

Reply:
Using attributes over direct color reference is also recommended by ODK's contribution guidelines.

Reply:
> do you find his arguments convincing?

not entirely but I'm not going to complain since it's a small thing. I just noticed the same case is with `colorAccent`, both colors are the same across both themes so adding them to both themes is a duplication I think but fair enough.